<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Individual</class>
 <widget class="QMainWindow" name="Individual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仪器设备管理(个人版)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NavBar" name="wg_guide" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="10,2">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>477</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="LunarCalendarItem" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="information">
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,8">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="2,2,2,2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="3,2">
                <item>
                 <widget class="QLabel" name="lab_username">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>国资部 张骁</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 170, 0);</string>
                  </property>
                  <property name="text">
                   <string>修改密码</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>在用设备数量:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lab_instrumentnumber">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>5</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="8,2,4">
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>记录:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lab_messagenumber">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>5</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="font">
                   <font>
                    <family>楷体</family>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>条</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="pb_diffrent">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 170, 255);</string>
                </property>
                <property name="text">
                 <string> 删除记录</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLineEdit" name="le_password_old">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>旧密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_password_new">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>新密码</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>明文</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_ok">
                <property name="text">
                 <string>确定</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_cancel">
                <property name="text">
                 <string>取消</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="NavProgress" name="wg_progreese" native="true"/>
        </item>
        <item>
         <widget class="QTableView" name="tableView_apply">
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="instrument">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,3,2,2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,8">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>设备条码:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lab_id">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,8">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>设备编号:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lab_bianhao">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,8">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>设备价格:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lab_price">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,8">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>设备名称:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lab_name">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,8">
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>设备型号:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lab_mode">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,8">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>入库时间:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lab_time">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pb_apply">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 170, 0);</string>
              </property>
              <property name="text">
               <string>申请设备</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pb_return">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 170, 0);</string>
              </property>
              <property name="text">
               <string>归还设备</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="ProgressPercent" name="wg_canzhi" native="true"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="tableView_instrument">
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavBar</class>
   <extends>QWidget</extends>
   <header location="global">navbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgressPercent</class>
   <extends>QWidget</extends>
   <header location="global">progresspercent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NavProgress</class>
   <extends>QWidget</extends>
   <header location="global">navprogress.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LunarCalendarItem</class>
   <extends>QWidget</extends>
   <header location="global">lunarcalendaritem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
